dev-util/lcov/lcov-1.12.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="A graphical front-end for GCC's coverage testing tool gcov"
+DESCRIPTION="collects gcov data for multiple source files and \
+creates HTML pages containing the source code annotated with \
+coverage information. It also adds overview pages for easy \
+navigation within the file structure. LCOV supports statement, \
+function and branch coverage measurement."
+HOMEPAGE="http://ltp.sourceforge.net/coverage/lcov.php"
+SOURCE_URI="https://github.com/linux-test-project/lcov/releases/download/v1.12/lcov-1.12.tar.gz"
+CHECKSUM_SHA256="b474e49c6c962754063b3be97a757a2ba9e7a455f0aea612863bf67e9b8b8ea7"
+COPYRIGHT="2002 - 2016 The lcov Developers"
+LICENSE="GNU GPL v2"
+REVISION="1"
+PATCHES="lcov-1.12.patchset"
+
+ARCHITECTURES="x86"
```

Comment:
Yes; but that's not really this's fault, that's GCC2's fault. So I say make it "any".